x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ilm/LifecycleSettings.java

```diff
@@ -27,6 +27,8 @@
     public static final String SLM_RETENTION_DURATION = "slm.retention_duration";
     public static final String SLM_MINIMUM_INTERVAL = "slm.minimum_interval";
 
+    public static final String SNAPSHOT_INDEX_NAME = "index.store.snapshot.index_name";
```

Comment:
Super minor, but can you add a comment about what this is for (since it's not actually an ILM setting, just one we check)